router/router_test.go

```diff
@@ -157,6 +157,57 @@ func initRouter() {
 	InitRouterAdmin()
 }
 
+func TestBackoff(t *testing.T) {
```

Comment:
By looking at these tests it is not clear to me how `minRetryBackoff` and `maxRetryBackoff` affect the results. Can we have them as parameters?

If I run the old algorithm with `10s` and `300s` for example I get the following:

```go
func TestBackoff(t *testing.T) {
	minRetryBackoff := 10 * time.Second
	maxRetryBackoff := 300 * time.Second

	b := backoff.NewExponentialBackOff()
	b.InitialInterval = minRetryBackoff
	b.MaxInterval = maxRetryBackoff
	b.RandomizationFactor = 0
	b.MaxElapsedTime = 0
	b.Multiplier = 2
	b.Reset()
	for index := 0; index < 10; index++ {
		d := b.NextBackOff()
		t.Logf("attempt: %d, duration: %s", index+1, d)
	}
}
```

Which gives:

```
=== RUN   TestBackoff
    backoff_test.go:23: attempt: 1, duration: 10s
    backoff_test.go:23: attempt: 2, duration: 20s
    backoff_test.go:23: attempt: 3, duration: 40s
    backoff_test.go:23: attempt: 4, duration: 1m20s
    backoff_test.go:23: attempt: 5, duration: 2m40s
    backoff_test.go:23: attempt: 6, duration: 5m0s
    backoff_test.go:23: attempt: 7, duration: 5m0s
    backoff_test.go:23: attempt: 8, duration: 5m0s
    backoff_test.go:23: attempt: 9, duration: 5m0s
    backoff_test.go:23: attempt: 10, duration: 5m0s
--- PASS: TestBackoff (0.00s)
```